tests/sentry/models/test_grouphash.py

```diff
@@ -92,3 +99,210 @@ def test_no_fingerprint(self) -> None:
         GroupHashMetadata.objects.create(grouphash=grouphash, hashing_metadata=hashing_metadata)
 
         assert grouphash.get_associated_fingerprint() is None
+
+
+class CacheInvalidationTest(TestCase):
```

Comment:
Nice tests. The only case I can think of that might be useful to add is:
- Verify that cache updates aren't made when caching is disabled via the `grouping.use_ingest_grouphash_caching` option

Reply:
This might also be good to explicitly include in the tests? From my comment above.

> Related: the `pre_save` signal runs during object creation as well.